install issue

**Describe the bug**

Failed to install from R 

**To Reproduce**
devtools::install_github("Sage-Bionetworks/mhealthtools")

**Screenshots**
```
clang -I/usr/local/Cellar/r/3.4.2/lib/R/include -DNDEBUG   -I/usr/local/opt/gettext/include -I/usr/local/opt/readline/include -I/usr/local/include   -fPIC  -g -O2  -c seewave_init.c -o seewave_init.o
clang -I/usr/local/Cellar/r/3.4.2/lib/R/include -DNDEBUG   -I/usr/local/opt/gettext/include -I/usr/local/opt/readline/include -I/usr/local/include   -fPIC  -g -O2  -c stft.c -o stft.o
stft.c:3:11: fatal error: 'sndfile.h' file not found
# include <sndfile.h> //Library for write or read sound file
          ^~~~~~~~~~~
1 error generated.
make: *** [stft.o] Error 1
ERROR: compilation failed for package ‘seewave’
* removing ‘/usr/local/lib/R/3.4/site-library/seewave’
Installation failed: Command failed (1)
'/usr/local/Cellar/r/3.4.2/lib/R/bin/R' --no-site-file --no-environ --no-save  \
  --no-restore --quiet CMD INSTALL  \
  '/private/var/folders/0k/5wwxg48s2kl93vk4nrh11pb40000gn/T/Rtmpry5W9g/devtools52f23751ed2f/Sage-Bionetworks-mhealthtools-8df1ea3'  \
  --library='/usr/local/lib/R/3.4/site-library' --install-tests

ERROR: dependency ‘seewave’ is not available for package ‘mhealthtools’
* removing ‘/usr/local/lib/R/3.4/site-library/mhealthtools’
```

**Desktop (please complete the following information):**
```
R version 3.4.2 (2017-09-28)
Platform: x86_64-apple-darwin16.7.0 (64-bit)
Running under: macOS High Sierra 10.13.5

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libLAPACK.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

loaded via a namespace (and not attached):
 [1] httr_1.3.1      compiler_3.4.2  R6_2.2.2        tools_3.4.2
 [5] withr_2.1.2     curl_3.2        memoise_1.1.0   knitr_1.20
 [9] git2r_0.21.0    digest_0.6.15   devtools_1.13.5
```

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ $ apt install libfftw3-3 libfftw3-dev libsndfile1 libsndfile1-dev
 
 to install these system dependencies, then retry the above `devtools` command.
 
-If you are still having issues installing `seewave`, it may be necessary to also install the `rgl` library system-wide.
+If you are still having issues installing `seewave`, it may be necessary to also install the `rgl` library.
 
 ```
 $ apt install r-cran-rgl

--- a/README.md
+++ b/README.md
@@ -12,13 +12,21 @@ First, install the `mhealthtools` package using `devtools`:
 devtools::install_github("Sage-Bionetworks/mhealthtools")
 ```
 
-A common issue when installing the `seewave` dependency is to be missing the system dependencies `libfftw3` and ` libsndfile1`. In a shell, run:
+A common issue when installing the `seewave` dependency on Linux systems is to be missing the system dependencies `libfftw3` and ` libsndfile1`. In a shell, run:
 
 ```
 $ apt install libfftw3-3 libfftw3-dev libsndfile1 libsndfile1-dev
 ```
 
 to install these system dependencies, then retry the above `devtools` command.
+
+If you are still having issues installing `seewave`, it may be necessary to also install the `rgl` library system-wide.
+
+```
+$ apt install r-cran-rgl
+```
+
+See the `seewave` installation page for more info: http://rug.mnhn.fr/seewave/
 
 ### Usage
 

--- a/README.md
+++ b/README.md
@@ -12,6 +12,14 @@ First, install the `mhealthtools` package using `devtools`:
 devtools::install_github("Sage-Bionetworks/mhealthtools")
 ```
 
+A common issue when installing the `seewave` dependency is to be missing the system dependencies `libfftw3` and ` libsndfile1`. In a shell, run:
+
+```
+$ apt install libfftw3-3 libfftw3-dev libsndfile1 libsndfile1-dev
+```
+
+to install these system dependencies, then retry the above `devtools` command.
+
 ### Usage
 
 

